resources/views/users/index.blade.php

```diff
@@ -12,6 +12,8 @@
             <th> {{ trans_choice('team.name',1) }}      </th>
             <th> {{ trans_choice('team.email',2) }}     </th>
             <th> {{ trans_choice('team.team',2) }}      </th>
+            <th> {{ trans('team.admin') }} {{ trans('team.role') }}     </th>
```

Comment:
Thank you so much for this request, I was thinking that maybe we can just migrate this page to use thrust too (like the ticket index) this way it will automatically have the sorting, searching and filtering features, and then we can easily create the actions to convert them to admin/assitant

Basically what you need to do is to create a Class that extends Resource called Agens and place it into App\Thrust

With that, you can access it by simply going go `handesk.test/thrust/agents` to do the testing, when everything works, in the agensController just need to update the index function to use return `(new ThrustController)->index('agents`)`

Let me know if something is not clear so I can help you better

Reply:
sounds good, i'll do that, thanks ;)